lib/shared/drive-constraints.js

```diff
@@ -276,6 +276,29 @@ exports.isDriveSizeRecommended = (drive, image) => {
   return _.get(drive, [ 'size' ], UNKNOWN_SIZE) >= _.get(image, [ 'recommendedDriveSize' ], UNKNOWN_SIZE)
 }
 
+/**
+ * @summary 64GB
+ * @private
+ * @constant
+ */
+exports.LARGE_DRIVE_SIZE = 64000000000
```

Comment:
Can we make this a factor, or use the e-expression, to improve clarity & readability – i.e. `64 * 1000 * 1000` or `64e6`?